Задания с 18.02.2022/Task1/Task1/BubbleSort.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+class Sort
+{
+    static void Swap(ref int a, ref int b)
```

Comment:
А для методов пишутся модификаторы видимости всегда, даже если видимость по умолчанию устраивает
```suggestion
    private static void Swap(ref int a, ref int b)
```